pytorch_lightning/trainer/lr_finder.py

```diff
@@ -321,8 +321,9 @@ def suggestion(self, skip_begin: int = 10, skip_end: int = 1):
 
         """
         try:
-            loss = self.results["loss"][skip_begin:-skip_end]
-            min_grad = (np.gradient(np.array(loss))).argmin()
+            loss = np.array(self.results["loss"][skip_begin:-skip_end])
+            loss = loss[~np.isnan(loss)]
```

Comment:
is perhaps [torch.isfinite]( https://pytorch.org/docs/stable/torch.html#torch.isfinite) a better option here? it would also include +/- inf values. What do you think?

Reply:
Yep. torch.isfinite is a better option here. Let me change it.

Reply:
nice. numpy works too :)